app/Http/routes.php

```diff
@@ -20,7 +20,9 @@
 #end of testing routes for dann
 
 # reports routing
-Route::get('report', 'ReportController@getReport');
+Route::get('/report/district/{districtId}', 'ReportController@getReportDistrictById');
+Route::get('/report/hood/{hoodId}', 'ReportController@getHoodById');
```

Comment:
@aniluyg `/report/district/` route seems to work, but hood currently produces an error; please comment or build hood functionality before we can merge